### PR TITLE
[15.0] FWD port of multiple PRs

### PR DESCRIPTION
**Pull request by etobella**
Small list of PRs pending to port to 15.0 :wink: 

https://github.com/OCA/edi/pull/639
https://github.com/OCA/edi/pull/651
https://github.com/OCA/edi/pull/652
https://github.com/OCA/edi/pull/653
https://github.com/OCA/edi/pull/645
https://github.com/OCA/edi/pull/648
https://github.com/OCA/edi/pull/662
https://github.com/OCA/edi/pull/666
https://github.com/OCA/edi/pull/672
https://github.com/OCA/edi/pull/694
https://github.com/OCA/edi/pull/696
https://github.com/OCA/edi/pull/697
https://github.com/OCA/edi/pull/698
https://github.com/OCA/edi/pull/721
https://github.com/OCA/edi/pull/762
https://github.com/OCA/edi/pull/764
https://github.com/OCA/edi/pull/763
https://github.com/OCA/edi/pull/765
https://github.com/OCA/edi/pull/767
https://github.com/OCA/edi/pull/774
https://github.com/OCA/edi/pull/778
https://github.com/OCA/edi/pull/797
https://github.com/OCA/edi/pull/790
https://github.com/OCA/edi/pull/808

### CODE REVIEW

**Comment by simahawk** [comment]
@etobella we must add a blacklist file for oca-port. I'll add one for v16.
You can use the new cli command on oca-port to achieve this quickly with `oca-port-pr blacklist OCA/edi#XX`.
I have a pending TODO for v16 too :wink:

**Review by bosd — APPROVED** [review]
TL;DR But it's green :green_circle:
We need to fix the tests on 15.0 branch. 
So :crossed_fingers: on this one

**Comment by etobella** [comment]
@simahawk shall we proceed with merge?

**Comment by pedrobaeza** [comment]
Wow, "small list of PRs" having 85 files changed. I wouldn't call it small...

Should I proceed with the merge?

**Review by simahawk — CHANGES_REQUESTED** [review]
@etobella IMO there are too many things here... included the migration of some modules, like https://github.com/OCA/edi/pull/814/commits/0f2904bb413546f107515352c065d41651cb6377

We cannot even bump properly the versions...

I don't think we need everything to fix the build.
I've just fixed the issue on v16 here https://github.com/OCA/edi-framework/pull/5/commits/f1e6976f1bc510bbb014be53c343ebcf0f80a88f and I will rework the fix for v14 and v15.

I'll check in a while

**Comment by simahawk** [comment]
As for the builds... there you go:

v14 https://github.com/OCA/edi/pull/822
v15 https://github.com/OCA/edi/pull/823

**Comment by MiquelRForgeFlow** [comment]
conflicts
